_includes/tutorials/joining-stream-table/confluent/markup/dev/ccloud-consume-rated-movies.adoc

```diff
@@ -0,0 +1,7 @@
+Before you start producing ratings, it's a good idea to set up the consumer on the output topic. This way, as soon as you produce ratings (and they're joined to movies), you'll see the results right away. Run this to get ready to consume the rated movies:
+
+```
```

Comment:
Similar comment as the other KT -- need step to tell user to enter SR creds?

Reply:
Same as the other PR, yes 100. I'll make sure my other PRs have this as well. Updated in commit https://github.com/confluentinc/kafka-tutorials/pull/827/commits/4619d631f56d6f894297fb2585ba93b3244647bf

Reply:
Thanks!